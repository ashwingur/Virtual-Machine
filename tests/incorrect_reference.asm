FUNC LABEL 0
    PRINT VAL 2
    REF VAL 5 VAL 8
    RET