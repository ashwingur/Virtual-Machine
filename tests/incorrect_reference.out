2
REF: The first agument must be a stack address and the second argument must be a stack symbol.
